public/app/percona/environment-overview/EnvironmentOverview.styles.ts

```diff
@@ -0,0 +1,10 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { spacing } }: GrafanaTheme2) => ({
+  widgetsWrapper: css`
+    display: grid;
```

Comment:
Let's try not to mix different layout paradigms unless strictly necessary. On PMM we usually go with flex.

Reply:
Yeah, it depends on the use case. The grid layout still isn't as much supported as flex still. If it's something we can achieve with flex, there's no need to compromise the layout by using grid. Many times a simple grid can be achieved with flex by just using margins on the children and negative margins on the parent.